ui/app/common/pnc-client/pagination/filteringPaginator.js

```diff
@@ -38,7 +38,7 @@
 
       var SEARCH_WILDCARD_CHAR = '*';
 
-      function filteringPaginator(page, search) {
+      function filteringPaginator(page, search, userId) {
```

Comment:
I added a new parameter to specify whether the paginator is created for a specific user

Reply:
Hi @DnsZhou  please add JSDoc comment describing all parameters for `filteringPaginator`